### DISCUSSION TITLE
Can HFS Layout Theme from Version 2.3m be used in Newer Versions

### DISCUSSION THREAD

**Discussion by kenrmayfield** [rejetto/hfs#119] [Ideas]
I just tried Version 2.4.0RC6 Release for the Log Out Feature which is really good.  I noticed there is a New Layout Theme.
Is there anyway to be able to use the Layout Theme from Version 2.3m with 2.4.0RC6?

Thanks,
Kenneth

**Comment by rejetto** [comment]
I suppose, yes. You can try to copy and paste, i guess. Never tried.

**Comment by kenrmayfield** [comment]
Massimo,

Would you know which Files to Copy?   Is it a Style Sheet?

Thanks,
Kenneth

On 2023-02-27 08:02 AM, Massimo Melina wrote:

> I suppose, yes. You can try to copy and paste, i guess. Never tried.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-5123155
[2]

**Comment by rejetto** [comment]
html template > edit
<img width="405" alt="image" src="https://user-images.githubusercontent.com/1367199/221586734-03bbbbe2-608e-4738-9087-2fef77904cd2.png">

**Comment by kenrmayfield** [comment]
Massimo,

So is this the Area were I take the Style Sheet or Code from 2.3m and 
Replace the Style Sheets in Newer Versions?

I do admit the 2.4.0RC6 Layout Theme was a different.  I just like the 
Layout Theme from 2.3m.

Thanks,
Kenneth

On 2023-02-27 08:15 AM, Massimo Melina wrote:

> html template > edit
> [1]
> 
> --
> Reply to this email directly, view it on GitHub [2], or unsubscribe 
> [3].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://user-images.githubusercontent.com/1367199/221586734-03bbbbe2-608e-4738-9087-2fef77904cd2.png
[2] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-5123321
[3]

**Comment by rejetto** [comment]
the easy way is to copy the whole code, otherwise it gets complicated

**Comment by kenrmayfield** [comment]
Massio,

Can you Explain a little?  Do you mean Copy All of the HTML Files from 
2.3m to 2.4.0RC6?

On 2023-02-27 08:38 AM, Massimo Melina wrote:

> the easy way is to copy the whole code, otherwise it gets complicated
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-5123566
[2]

**Comment by rejetto** [comment]
1. you click where i told you, html template > edit
2. you select the whole text, copy it
3. go to the new version, edit again, and paste

**Reply by PH5HP** [reply]
Hi Rejetto,

Where exactly can we paste that in the new version? I am searching for it but can't find it yet.

Kind regards, Henk

**Reply by PH5HP** [reply]
I think I found it :-) 

I like this new approach (new version), but I hope that the download and other statistics will be added in the future...

Kind regards, Henk

**Comment by kenrmayfield** [comment]
Massimo,

I just tried Pasting the HTML Code from 2.3m to 2.4rc7 and it states 
that the "The Template your trying to Load is not Compatible with 
Current HFS".

Wish Version 2.3m had the Logout Feature.

2.3m Layout I really like.   The other Layouts are nice as well but 2.3m 
Layout has a more Work Office Look.

Kenneth

On 2023-02-28 05:49 PM, Massimo Melina wrote:

> * you click where i told you, html template > edit
> * you select the whole text, copy it
> * go to the new version, edit again, and paste
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-5160178
[2]

**Comment by kenrmayfield** [comment]
Massimo,

I was checking back with you on my Previous Message.

Hope you had a good weekend?

Thanks,
Kenneth

On 2023-03-01 08:48 PM, ***@***.*** wrote:

> Massimo,
> 
> I just tried Pasting the HTML Code from 2.3m to 2.4rc7 and it states 
> that the "The Template your trying to Load is not Compatible with 
> Current HFS".
> 
> Wish Version 2.3m had the Logout Feature.
> 
> 2.3m Layout I really like.   The other Layouts are nice as well but 
> 2.3m Layout has a more Work Office Look.
> 
> Kenneth
> 
> On 2023-02-28 05:49 PM, Massimo Melina wrote:
> 
>> * you click where i told you, html template > edit
>> * you select the whole text, copy it
>> * go to the new version, edit again, and paste
>> 
>> --
>> Reply to this email directly, view it on GitHub [1], or unsubscribe 
>> [2].
>> You are receiving this because you authored the thread.Message ID: 
>> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-5160178
[2]

**Comment by rejetto** [comment]
I don't remember what it takes to make a 2.3 template compatible with 2.4, years have passed and version 3's information has overwritten most of it.
You should study both of the templates (2.3 and 2.4) to understand what's to be added, and it takes some time.
If you don't find enough help here you can try https://rejetto.com/forum/index.php?board=21.0

If you know CSS you can try to customize the look of version 3 instead.

**Comment by kenrmayfield** [comment]
Massimo,

Checking back too see if there is any other Idea on making HTML Code 
from 2.3m Copied to 2.4rc7 to work?

It states that the "The Template your trying to Load is not Compatible 
with Current HFS".

On 2023-03-06 1:19 am, ***@***.*** wrote:

> Massimo,
> 
> I was checking back with you on my Previous Message.
> 
> Hope you had a good weekend?
> 
> Thanks,
> Kenneth
> 
> On 2023-03-01 08:48 PM, ***@***.*** wrote:
> 
> Massimo,
> 
> I just tried Pasting the HTML Code from 2.3m to 2.4rc7 and it states 
> that the "The Template your trying to Load is not Compatible with 
> Current HFS".
> 
> Wish Version 2.3m had the Logout Feature.
> 
> 2.3m Layout I really like.   The other Layouts are nice as well but 
> 2.3m Layout has a more Work Office Look.
> 
> Kenneth
> 
> On 2023-02-28 05:49 PM, Massimo Melina wrote:
> 
> * you click where i told you, html template > edit
> * you select the whole text, copy it
> * go to the new version, edit again, and paste
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-5160178
[2]

**Comment by rejetto** [comment]
that message is convenient in the sense that it's stopping you from further useless attempts.
No news on top of what I already suggested.
I guess you don't know css, because that's what it takes to customize the look of current version, but you may find someone to do the job for you.
In case, consider this guide for HFS 3 https://github.com/rejetto/hfs/wiki/customization

**Comment by kenrmayfield** [comment]
Henk,

Can you please Explain where you Found  how to Replace the eNew Version 
Theme with 2.3m Theme?

How you did it also?

Kenneth

On 2023-07-30 7:16 am, PH5HP wrote:

> I think I found it :-)
> 
> I like this new approach (new version), but I hope that the download 
> and other statistics will be added in the future...
> 
> Kind regards, Henk
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-6587046
[2]

**Reply by PH5HP** [reply]
Hi Kenneth,

Sorry but that was not what I ment to say. I found the Custom HTML section where I can do the alterations to the theme. Simply copy and paste did not work for the theme I used in version 2.3. 

If you you want to use that one I think it will be diffecult or maybe impossible. My advice to you would be to alter the default one in version 3 by using som CSS and HTM in the Custom HTML section.

Kind regards,
Henk - PH5HP - http://ftp.ph5hp.nl

**Comment by kenrmayfield** [comment]
Henk,

Ok.......I was hoping.    The 2.3 Theme looks better in a Corporate 
Environment.   It looks more Business like.

Users really like the 2.3 Theme better.  With the Latest Version of HFS 
and the 2.3 Theme.......that would be
really great if that was possible.

  Kenneth
On 2023-07-31 4:59 am, PH5HP wrote:

> Hi Kenneth,
> 
> Sorry but that was not what I ment to say. I found the Custom HTML 
> section where I can do the alterations to the thee. Simply copy and 
> paste did not work for the theme I used in version 2.3.
> 
> If you you want to use that one I think it will be diffecult or maybe 
> impossible. My advice to you would be to alter the default one in 
> version 3 by using som CSS and HTM in the Custom HTML section.
> 
> Kind regards,
> Henk - PH5HP - http://ftp.ph5hp.nl
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-6593375
[2]

**Comment by rejetto** [comment]
replicating the whole thing can take a bit.
Can you point at 2-3 things that you'd like differently?

**Comment by kenrmayfield** [comment]
Massimo,

The Whole Look of the Theme from 2.3 Version would be Nice as a Option 
to use with the New Version of HFS.
Any Forecast on Active Directory Integration?

Below is the Theme from 2.3:

On 2023-07-31 11:38 am, Massimo Melina wrote:

> replicating the whole thing can take a bit.
> Can you point at 2-3 things that you'd like differently?
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-6597284
[2]

**Comment by rejetto** [comment]
maybe you replied via email and an attachment went lost?
and btw, you didn't answer my question ;-)
no forecast on AD yet, I don't even have an idea of what the effort would be.

**Comment by kenrmayfield** [comment]
Here it is again.  Look at Previous Email below.

On 2023-07-31 6:08 pm, ***@***.*** wrote:

> Massimo,
> 
> The Whole Look of the Theme from 2.3 Version would be Nice as a Option 
> to use with the New Version of HFS.
> Any Forecast on Active Directory Integration?
> 
> Below is the Theme from 2.3:
> 
> On 2023-07-31 11:38 am, Massimo Melina wrote:
> 
>> replicating the whole thing can take a bit.
>> Can you point at 2-3 things that you'd like differently?
>> 
>> --
>> Reply to this email directly, view it on GitHub [1], or unsubscribe 
>> [2].
>> You are receiving this because you authored the thread.Message ID: 
>> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/119#discussioncomment-6597284
[2]

**Comment by rejetto** [comment]
what i meant was that attachments via email are not supported, or so it seems

**Comment by kenrmayfield** [comment]
Massimo,

The Whole Look of the Theme from 2.3 Version would be Nice as a Option
to use with the New Version of HFS.
 
Below is the Theme from 2.3:

![image](https://github.com/rejetto/hfs/assets/89940641/29bb3ee1-577e-4429-b6fc-2f2ef6d25fa9)

**Reply by PH5HP** [reply]
Hi,

I agree about the extra information in the theme. I would like some statistics beeing shown in the theme also. See screenshot below. It would be nice to have tokens which could be used to add them in the layout.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl
![image](https://github.com/rejetto/hfs/assets/140949730/8fdddb24-65bb-4f5f-b423-c6b52e06eb14)

**Comment by kenrmayfield** [comment]
@rejetto 

Massimo,

I was checking back with you on the below response:


The Whole Look of the Theme from 2.3 Version would be Nice as a Option
to use with the New Version of HFS.
 
Below is the Theme from 2.3:

![image](https://github.com/rejetto/hfs/assets/89940641/29bb3ee1-577e-4429-b6fc-2f2ef6d25fa9)

**Comment by rejetto** [comment]
hi Kenneth, if you are asking me if I'm going to produce a theme like that, it's not currently in my plans.
And yet, the system is open and people with web-authoring skills can make it, and publish it, freely, without my intervention.
For the moment only one theme was published. Hopefully more will come.

**Comment by kenrmayfield** [comment]
Massimo,

Ok....Understand.

I thought it was a easy integration of the 2.3m Theme to be used with 
the Latest or Newest Version of HFS File Server.

**Comment by rejetto** [comment]
yeah, it is actually something to be made from scratch

**Comment by kenrmayfield** [comment]
Massimo,

If was possible for that 2.3m Template,  alot of Business's really do 
like that Theme.

Reason being that it Basic, Informative, Windows File Manger Look(which 
most Users are use too) and more Business looking.

**Comment by rejetto** [comment]
I'm sorry but I'm already maxed out with the many missing features I'm working on

**Comment by PH5HP** [comment]
Hi Massimo,

How about some brief guide with information what to change if I want to make my own layout? Classes names, color names etc.

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
hi Henk,
I started this guide that can be expanded https://github.com/rejetto/hfs/wiki/Customization
But it's almost impossible to make a compelte list of possibilities, because they are endless. 
The page is editable by anybody and everyone is welcome to expand it.
Styling allows too many things, even tho there are limits to what you can achieve without changing the structure of the html.

The practical way is (given some knowledge of web authoring) to right-click on what you want to change and "inspect element".
If you don't know this tool, you can have a look at some tutorial https://youtu.be/oRKlKhFt2Rg
With it, you get all the information on that part, plus the ability to experiment with changes without saving. When you are satisfied with your changes, you can report them inside HFS configuration.
Maybe we should have this info on the wiki page.

**Comment by kenrmayfield** [comment]
@rejetto 


Is there still a way to get the HFS 2.3m Theme Layout and use it with Version 3?<br><br>

### 2.3m THEME:
<img width="698" height="511" alt="image" src="https://github.com/user-attachments/assets/6e247ef5-2b37-4c7f-8ca3-d39c63580a61" />

**Reply by rejetto** [reply]
if you mean the same look, i'm not sure, but maybe yes. You will know only by trying.

**Comment by kenrmayfield** [comment]
@rejetto 

Your the Developer.  

How can I get a Copy of the HTML for 2.3m?
